<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CouponWidget</class>
 <widget class="QDialog" name="CouponWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="couponTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="allCouponBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unusedCouponBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="usedCouponBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="expiredCouponBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="useCouponBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteCouponBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
